org.thingpedia.iot.tv/manifest.tt

```diff
@@ -0,0 +1,58 @@
+abstract class @org.thingpedia.iot.tv
+#_[thingpedia_name="TV"]
+#_[thingpedia_description="Interface for TV."]
+#_[description="Interface for TV."]
+#[license="BSD-3-Clause"]
+#[license_gplcompatible=true]
+#[subcategory="home"]
+{
+  monitorable query state(out state : Enum(on,off))
+  #_[canonical="switch state"]
+  #_[confirmation="the state of $__device"]
+  #_[formatted=["It is ${state}"]];
```

Comment:
I know you're fond of just using `state`, but in an interface with a lot of functions, it's unclear what `state` is. Also, copy-paste errors.

Reply:
Changed in devices other than most sensors and vacuum.